src/app/Dashboard/Dashboard.tsx

```diff
@@ -270,9 +270,9 @@ export const Dashboard: React.FC<DashboardProps> = (_) => {
 
   return (
     <TargetView pageTitle="Dashboard" compactSelect={false}>
-      <Grid hasGutter>
+      <Grid id={'dashboard-grid'} hasGutter>
         {cardConfigs.map((cfg, idx) => (
-          <GridItem span={cfg.span} key={idx}>
+          <GridItem span={cfg.span} key={idx} order={{ default: idx.toString() }}>
```

Comment:
I think we should start using unique keys now. Example issue: Try reordering an AA card (i.e. drag or replace it with other card), you will notice this AA card is being freshly built (i.e. one-time subscription to context.target.authRetry is sub again).

I think this is because React looks at `GridItem` keys and determine that they are the same instances (keys are indexs - reorder does not change). However, children under those GridItem are not different -> unmount entire children + freshly rebuild.

Reply:
We have a dependency on `nanoId` which can use in company with card type name to solve this (need to include in redux store state)